components/FederalBanner.js

```diff
@@ -1,43 +1,59 @@
 import React from 'react'
 import { css } from 'react-emotion'
+import styled from 'react-emotion'
```

Comment:
You can shorten this import string by combining it with the one above it.
eg: 
`import styled, { css } from 'react-emotion'`